tests/foreman/cli/test_oscap.py

```diff
@@ -128,13 +155,19 @@ def test_positive_view_scap_content_info_admin(self):
 
         :expectedresults: The info of the scap-content is listed.
 
-        :caseautomation: notautomated
+        :caseautomation: automated
 
         :CaseImportance: Critical
         """
+        title = gen_string('alpha')
+        make_scapcontent({'title': title, 'scap-file': '/tmp/{0}'.format(
+                            self.file_name)})
```

Comment:
nitpick: If 2 dict keys do not fit in one line just use different style with 1 item per line:
```python
        make_scapcontent({
            'title': title,
            'scap-file': '/tmp/{0}'.format(self.file_name),
        })
```
It's much more readable.

Reply:
Applicable for other tests as well